bob/src/cluster/quorum.rs

```diff
@@ -125,7 +126,8 @@ impl Quorum {
         key: BobKey,
         data: &BobData,
         at_least: usize,
-    ) -> (Tasks<Error>, Vec<NodeOutput<Error>>) {
+        affected_replicas_by_node: &HashMap<NodeName, usize>
+    ) -> (Tasks<RemotePutResponse, RemotePutError>, Vec<NodeOutput<RemotePutError>>) {
```

Comment:
It is better to add successful results as well here:
```suggestion
    ) -> (Tasks<RemotePutResponse, RemotePutError>, Vec<NodeOutput<RemotePutResponse>>, Vec<NodeOutput<RemotePutError>>) {
```